src/remote/configuration/__init__.py

```diff
@@ -19,32 +19,32 @@ class RemoteConfig:
 
 
 @dataclass
-class SyncIgnores:
-    """Patterns used to ignore files when syncing with remote location"""
+class SyncRules:
+    """Patterns used to either ignore or exclude files when syncing with remote location"""
```

Comment:
> either ignore or exclude

Isn't it the same from rsync's point of view. I think it should be rephrased as `patterns used by rsync to forcefully exclude or include files while syncyng the with remote location`. Same applies to comments below

Reply:
Done.